### PR TITLE
Feat: Add Cypress File Upload Script

### PR DESCRIPTION
**Pull request by krisrjohnson21**
This PR adds an automated test ensuring that the Youth Pass file upload fields work as intended. 

Asana ticket: https://app.asana.com/0/1170867711449497/1200759639068379/f

### CODE REVIEW

File: cypress/automated-tests/youth-pass/successful-youth-pass-submission.spec.js [hunk 1]
```diff
@@ -39,18 +39,18 @@ describe('Youth Pass Successful Submission', () => {
         
         cy.get('#element114_Option_1').click().blur();
         cy.get('#element133').attachFile('youth-pass-test-image.png')
-        cy.get('.k-file-validation-message').should('exist');
+        cy.get('.k-text-success').should('exist');
```
Comment (krisrjohnson21) [inline cypress/automated-tests/youth-pass/successful-youth-pass-submission.spec.js:42]: While developing the file upload script, I realized that the `file-validation-message` class appears regardless of file success/failure. The updated `text-success` class provides a stronger assertion here.

File: cypress/automated-tests/youth-pass/file-type-upload.spec.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            cy.get('.k-upload-status > .k-button').click();
+            cy.get('.k-text-error').should('not.exist');
+        })
+
+        cy.get('#element39').attachFile('youth-pass-test-image.pdf')
+        cy.get('.k-text-success').should('exist');
+        cy.get('#form-section-8 > .form-section-buttons > .form-section-next').click();
+    });
+
+    it('uploads rtf and xlsx file types to proof of program enrollment', () => {
```
Comment (arkadyan) [inline cypress/automated-tests/youth-pass/file-type-upload.spec.js:97]: If I'm reading right, I think this test description should probably say something about not accepting these types of uploads, right?

**Review by arkadyan — APPROVED** [review]
In addition to my one small description suggestion, the other issue is that the `.DS_Store` should not be committed, and probably should get added to `.gitignore`. Other than those 2 items this looks great and should be ready to go.

File: cypress/automated-tests/youth-pass/file-type-upload.spec.js (hunk 2 above)
Comment (krisrjohnson21) [reply cypress/automated-tests/youth-pass/file-type-upload.spec.js:97]: Good call. I'm not quite sure how the `.DS_Store` snuck in there, I'll remove it and update the `it` blocks to more accurately describe which file types should and should not upload successfully

**Review by arkadyan — APPROVED** [review]

**Pull request merged**